frontend/src/metabase/browse/containers/TableBrowser/TableBrowser.unit.spec.js

```diff
@@ -36,11 +35,14 @@ describe("TableBrowser", () => {
       [{ id: 123, name: "foo", initial_sync_status: "complete" }],
       { overwriteRoutes: true },
     );
-    // advance the timer
-    jest.advanceTimersByTime(RELOAD_INTERVAL);
-    // wait for the response
-    const calls2 = fetchMock.calls(/\/api\/database\/1\/schema\/public/);
-    expect(calls2.length).toBe(2);
+    await waitFor(
```

Comment:
removed the fake timers here and gave `waitFor` enough time to retry the call. Alternatively, maybe we can remove the `waitFor` on line 24 if we want this to use fake timers